apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/enrollment/api/EnrollmentController.java

```diff
@@ -46,4 +46,14 @@
                 .collect(Collectors.toList());
         return ResponseEntity.ok(enrollments);
     }
+
+    @ApiOperation(value = "Get enrollment QR code", nickname = "get enrollment QR code", notes = "",
+            authorizations = {@Authorization(value = "JWT")})
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "If valid credentials were provided", response = Iterable.class),
+            @ApiResponse(code = 400, message = "If invalid data was provided")})
+    @GetMapping("{id}/qr-code")
+    ResponseEntity<byte[]> getQRCode(@PathVariable Long id) {
+        return ResponseEntity.ok(enrollmentService.generateQRCode(id));
```

Comment:
I think maybe a content type header should be added to the response, so the browser can interpret it as an image. Does swagger show a weird text in response or an image? With content type it shoud be an image that can be downloaded.

Reply:
Done